packages/desktop-client/src/components/manager/ConfigServer.tsx

```diff
@@ -127,37 +127,35 @@ export function ConfigServer() {
         </Text>
       )}
 
-      <form
-        style={{ display: 'flex', flexDirection: 'row', marginTop: 30 }}
-        onSubmit={e => {
-          e.preventDefault();
-          onSubmit();
-        }}
-      >
+      <View style={{ display: 'flex', flexDirection: 'row', marginTop: 30 }}>
         <BigInput
           autoFocus={true}
           placeholder="https://example.com"
           value={url || ''}
           onChangeValue={setUrl}
           style={{ flex: 1, marginRight: 10 }}
+          onEnter={onSubmit}
         />
         <ButtonWithLoading
-          type="primary"
-          loading={loading}
+          variant="primary"
+          aria-label="OK"
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
```